test/e2e/lib/browser-manager.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file Manages instance of Playwright Browser and BrowserContext.
+ * @author Edwin Takahashi
+ */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+import playwright from 'playwright';
+
+const browserStartTimeoutMS = 2000;
+
+export let browser;
+export const isPlaywright = true;
```

Comment:
Is there any need for this, if it is always true and doesn't get changed?

Reply:
The intention of this value was to be used outside of the `browser-manager` to switch codepaths depending on if Playwright behavior was needed, or legacy Selenium path was needed.

With that said, there has been feedback from Sergio in the follow-on PR (#51209) and he raised an interesting point that `magellan` has the ability to load specific hook files which would remove the need for me to have these flags in the first place.

Reply:
I've removed the `isPlaywright` flag in preparation for the next patch which will separate out mocha-hooks into Playwright and Selenium versions therefore rendering this flag moot.